FunctionSignatureMap.php

```diff
@@ -5978,13 +5980,13 @@
 'imap_utf8' => ['string', 'mime_encoded_text'=>'string'],
 'imap_utf8_to_mutf7' => ['string|false', 'in'=>'string'],
 'implode' => ['string', 'glue'=>'string', 'pieces'=>'array'],
-'implode\'1' => ['string', 'pieces'=>'array', 'glue='=>'string'],
```

Comment:
This was/was going to be added to the deltas for 8.0 - phan warns about this separately